src/org/labkey/filetransfer/view/fileTransferConfig.jsp

```diff
@@ -52,115 +47,149 @@
                     + 'onmouseout="return hideHelpDivDelay();"><span class="labkey-help-pop-up">?</span></a>';
         };
 
-        var containingFilesHeader = Ext4.create('Ext.form.Label', {
-            text: 'Files Directory Path',
-            style: 'font-weight: bold;'
+        var clientDataHeader = Ext4.create('Ext.form.Label', {
+            text: 'Client Registration Data',
+            style: 'font-weight: bold; font-size:large'
         });
 
-        var endpointField = Ext4.create('Ext.form.field.Text', {
-            name: "endpointPath",
-            labelWidth: 150,
-            width: 510,
-            padding: '10px 0 25px 0',
+        var clientId = Ext4.create('Ext.form.field.Text', {
+            name: "clientId",
+            labelWidth: 200,
+            width: 535,
+            padding: '10px 0 0 25px',
             hidden: false,
             disabled: false,
```

Comment:
I don't have this branch pulled down locally to look at the UI, but isn't "false" the default for the hidden and disabled properties of the Ext4 form elements? If so, can probably remove from all of the items below. I don't see anywhere that updates the hidden or disabled state of an item after it is initialized. Probably an artifact of the previous set of form items.